pyquil/api/_qpu.py

```diff
@@ -193,7 +185,7 @@ def get_result(self, execute_response: QPUExecuteResponse) -> QAMExecutionResult
         Retrieve results from execution on the QPU.
         """
 
-        async def _get_result(*args):
+        async def _get_result(*args) -> qcs_sdk.ExecutionResults:  # type: ignore
```

Comment:
lets test this as well

Reply:
removed for the same reason: https://github.com/rigetti/pyquil/pull/1528#discussion_r1106366340